compiler/hash-reporting/src/highlight.rs

```diff
@@ -0,0 +1,97 @@
+use std::ops::BitOr;
```

Comment:
How come the reporting stuff is here? Didn't we merge it to main before?

Reply:
No, I don't believe it was merged into main, only into `ast-gen`